atoma-sui/src/client.rs

```diff
@@ -1267,9 +1258,9 @@ pub(crate) mod utils {
     /// # Implementation Notes
     ///
     /// - The function queries up to 100 objects at a time
-    /// - The function filters objects by package and looks for the specific NodeBadge type
-    /// - Object content is parsed to extract the small_id from the Move object's fields
-    pub(crate) async fn get_node_badge(
+    /// - The function filters objects by package and looks for the specific `NodeBadge` type
+    /// - Object content is parsed to extract the `small_id` from the Move object's fields
+    pub async fn get_node_badge(
```

Comment:
Why do these methods have more permissive visibility than the method to which they are constrained ?

Reply:
Writing `pub(crate)` is misleading when it’s redundant due to the parent module’s visibility. You can read more [here](https://rust-lang.github.io/rust-clippy/master/index.html#redundant_pub_crate)

Reply:
Good to know !